code/code/misc/spells.h

```diff
@@ -322,6 +322,7 @@ enum spellNumT {
      SPELL_NUMB_DEIKHAN,
      SPELL_EARTHQUAKE_DEIKHAN,
      SPELL_CALL_LIGHTNING_DEIKHAN,
+     SPELL_MARTIAL_MIGHT,
```

Comment:
@cizra Here's the enum I was worried this might break something here. Sure it looks ok?